src/main/java/MockClass.java

```diff
@@ -5,4 +5,27 @@ public int doSomething(int seed) {
 
         return seed + 5;
     }
+
+    public static void main(String[] args) {
+
+        Day day = null;
+
+        switch (day) {
+            case MONDAY:
+            case TUESDAY:
+                WEDNESDAY:   // Noncompliant; syntactically correct, but behavior is not what's expected
+                doSomething();
+                break;
+        }
+    }
+
+    private static void doSomething() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3AS1186)